cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/FileLocalizer.java

```diff
@@ -64,17 +65,16 @@ public void run(String[] args) throws Exception {
 
     // Localize the runtime config jar
     URI uri = URI.create(args[0]);
-    Location runtimeConfigLocation;
-    if (context.getLocationFactory().getHomeLocation().toURI().getScheme().equals(uri.getScheme())) {
-      runtimeConfigLocation = context.getLocationFactory().create(uri);
-    } else {
-      runtimeConfigLocation = new LocalLocationFactory().create(new File(uri).toURI());
-    }
+    Location runtimeConfigLocation = new LocalLocationFactory().create(new File(uri.getPath()).toURI());
```

Comment:
Why the above if check is removed? If the `uri` is already referring to a local file, it shouldn't need to fetch from the masterEnv?

Reply:
Revived the code 

    if (context.getLocationFactory().getHomeLocation().toURI().getScheme().equals(uri.getScheme())) {
      runtimeConfigLocation = context.getLocationFactory().create(uri);

The code only checks LocationFactory and uri share the same filesystem scheme. They could both refer to the same non-local file system, right?